chia/util/logging.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+import logging
+import re
+import time
+from typing import Pattern
+
+
+class TimedDuplicateFilter(logging.Filter):
+    last_log_time: float
+    regex: Pattern[str]
+    min_time_wait_secs: int
+
+    def __init__(self, regex_str: str, min_time_wait_secs: int, name: str = ""):
```

Comment:
This forces a wait time of `min_time_wait_secs` between similar messages. Would this be more user-friendly as a rate (e.g. X msgs per minute) ?